pkg/controllers/resources/nodes/translate.go

```diff
@@ -26,8 +26,10 @@ var (
 func (s *nodeSyncer) translateUpdateBackwards(pNode *corev1.Node, vNode *corev1.Node) {
 	// merge labels & taints
 	translatedSpec := pNode.Spec.DeepCopy()
-	excludeAnnotations := []string{TaintsAnnotation, RancherAgentPodRequestsAnnotation, RancherAgentPodLimitsAnnotation}
-	labels, annotations := translate.ApplyMetadata(pNode.Annotations, vNode.Annotations, pNode.Labels, vNode.Labels, excludeAnnotations...)
+	applyMetadataOptions := translate.ApplyMetadataOptions{
+		ExcludeAnnotations: []string{TaintsAnnotation, RancherAgentPodRequestsAnnotation, RancherAgentPodLimitsAnnotation},
+	}
+	translate.SyncHostMetadataToVirtual(pNode, vNode, applyMetadataOptions)
```

Comment:
`translate.SyncHostMetadataToVirtual(...)` calls `translate.ApplyMetadata(...)` and also sets annotations and labels, which is why the code on the left side has been removed.